### ISSUE TITLE
Create updated tag / release?

### ISSUE THREAD

**Issue by ralphwetzel** [njh/node-red-contrib-owfs#26]
Hi @njh!
Do you mind to create a new tag / release a new version ... to make available the changes implemented in the last year(s)?
Thank you in advance for your effort!
R.

**Comment by njh** [comment]
Yes, would really like to get a new release out. Will try and do it within the next week.

**Comment by vicary** [comment]
That's really nice, can't wait for the update!

**Comment by ralphwetzel** [comment]
Any update on this?
@njh Is there an option to support you to get the new version out?

**Comment by XueSheng-GIT** [comment]
Just started with owfs and node-red and realised that some of the latest/important fixes are missing in the published version but are available in github. @njh any chance to get an updated version pushed to node-red repo?
